torchtitan/config_manager.py

```diff
@@ -212,6 +212,11 @@ def __init__(self):
             action="store_true",
             help="Whether to compile the model",
         )
+        self.parser.add_argument(
```

Comment:
IMO we should not add another option. We can just possibly reuse the `fused_rmsnorm` field in the norm_type, instead of adding a separate option here